src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -102,8 +102,11 @@ public static void MapText(IEditorHandler handler, IEditor editor)
 		public static void MapTextColor(IEditorHandler handler, IEditor editor) =>
 			handler.PlatformView?.UpdateTextColor(editor);
 
-		public static void MapPlaceholder(IEditorHandler handler, IEditor editor) =>
+		public static void MapPlaceholder(IEditorHandler handler, IEditor editor)
+		{
 			handler.PlatformView?.UpdatePlaceholder(editor);
+			handler.PlatformView?.UpdateCharacterSpacing(editor);
```

Comment:
Is there a way we could avoid calling this from here, and instead ensure from elsewhere that CharacterSpacing gets updated after?

Why is it that the existing MapCharacterSpacing from which the update happens isn't enough?

Reply:
Because in the `UpdateCharacterSpacing` extension method we make the changes to both `AttributedText` and `AttributedPlaceholderText`. When updating CharacterSpacing if it hasn't already been set the placeholder, AttributedPlaceholderText is null and will not be updated. That's why I added this line to update after initializing the placeholder.

We can improve this by splitting the UpdateCharacterSpacing extension method in two, although that would be a breaking change (public APIs).

Reply:
I wonder if this is a case for us to call `handler.UpdateValue` to get the correct mappers to fire:

```cs
handler.
```suggestion
			handler.UpdateValue(nameof(IEditor.CharacterSpacing));
```

@PureWeen was this the final form we found was good for all other mapper overrides?

Reply:
Right, could just use `handler.UpdateValue` to do the same. Changed.